### DISCUSSION TITLE
I have no image on my banner on main menu

### DISCUSSION THREAD

**Discussion by Yokomi8** [ppy/osu#21788] [Q&A] [answered]
![osu](https://user-images.githubusercontent.com/54288174/209479921-206f0a18-5307-4808-a63a-e856e3a8564e.png)

Vertion:2022.08.0

**Comment by frenzibyte** [comment]
Can you view banners of other users in the game? I've checked your profile and I can view it in the user profile overlay and as a card in the friends dashboard.

**Reply by Yokomi8** [reply]
i can see my friend banner

**Comment by peppy** [comment, marked as answer]
Need [logs](https://github.com/ppy/osu/discussions/15188).

**Reply by Yokomi8** [reply]
[network.log](https://github.com/ppy/osu/files/10305442/network.log)
[performance.log](https://github.com/ppy/osu/files/10305443/performance.log)
[runtime.log](https://github.com/ppy/osu/files/10305444/runtime.log)
[updater.log](https://github.com/ppy/osu/files/10305445/updater.log)
[database.log](https://github.com/ppy/osu/files/10305446/database.log)
[input.log](https://github.com/ppy/osu/files/10305447/input.log)
[legacy-ipc.log](https://github.com/ppy/osu/files/10305448/legacy-ipc.log)

**Reply by bdach** [reply]
Not sure what's happening here. It looks like your cover image was never requested by the game at all.

I've tried opening your profile locally and the cover image works fine there.

**Reply by peppy** [reply]
Were those logs from after you attempted to view your banner?

**Reply by Yokomi8** [reply]
That was my logs after view my banner
so... I don't know what to do for you

**Comment by cheeezie** [comment]
If it stays the same way after a restart, I'd suggest trying to change the banner to something else to see if changes take effect. And yes, send [Logs](https://github.com/ppy/osu/discussions/15188) so the devs can find the issue.
P.S. can you see your banner when you click on your profile?

**Reply by Yokomi8** [reply]
same problem, i change my banner for a .png or .jpeg doesn't work (and for your 2nd question i can see my banner on my profile.)
